SAFARI/README.md

```diff
@@ -0,0 +1,20 @@
+# Using BCA GPA on Safari
```

Comment:
Make this a section of the `README.md` in the repository's root, no need for a separate Safari folder/readme.

Reply:
Done